sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/block/flow/FlowRuleManager.java

```diff
@@ -83,7 +85,7 @@ public static void register2Property(SentinelProperty<List<FlowRule>> property)
      */
     public static List<FlowRule> getRules() {
         List<FlowRule> rules = new ArrayList<FlowRule>();
-        for (Map.Entry<String, List<FlowRule>> entry : flowRules.entrySet()) {
+        for (Map.Entry<String, List<FlowRule>> entry : flowRules.get().entrySet()) {
```

Comment:
Is it necessary to initial its value (flowRules) to be an empty map obviously?
It's tricky now because the initial empty value is initialized by `currentProperty.addListener(LISTENER);` in static block. It may trigger a bug if anyone change any of them in future.

Reply:
Totally agree.